test/load/locust-files/fleet_autoscaling.py

```diff
@@ -0,0 +1,214 @@
+              "scheduling":"Packed",
+              "strategy":{
+                  "type":"RollingUpdate"
+              },
+              "template":{
+                  "spec":{
+                      "ports":[
+                          {
+                              "name":"default",
+                              "portPolicy":"dynamic",
+                              "containerPort":26000
+                          }
+                      ],
+                      "template":{
+                          "spec":{
+                              "containers":[
+                                  {
+                                      "name":"simple-udp",
+                                      "image":"gcr.io/agones-images/udp-server:0.5"
+                                  }
+                              ]
+                          }
+                      }
+                  }
+              }
+          }
+      }
+      response = self.client.put(selfLink, data=json.dumps(payload), headers=headers)
+      self.waitForScaling(selfLink, fleet_size)
+      total_time = int((time.time() - start_time) * 1000)
+      events.request_success.fire(request_type="fleet_scaling_up", name="fleet_scaling_up", response_time=total_time, response_length=0)
+
+      # Scale down the fleet.
+      resource_version = self.getResourceVersion(selfLink)
+      start_time = time.time()
+      payload = {
+          "apiVersion":"stable.agones.dev/v1alpha1",
+          "kind":"Fleet",
+          "metadata":{
+              "name":str(name),
+              "namespace":"default",
+              "resourceVersion":str(resource_version)
+          },
+          "spec":{
+              "replicas":0,
+              "scheduling":"Packed",
+              "strategy":{
+                  "type":"RollingUpdate"
+              },
+              "template":{
+                  "spec":{
+                      "ports":[
+                          {
+                              "name":"default",
+                              "portPolicy":"dynamic",
+                              "containerPort":26000
+                          }
+                      ],
+                      "template":{
+                          "spec":{
+                              "containers":[
+                                  {
+                                      "name":"simple-udp",
+                                      "image":"gcr.io/agones-images/udp-server:0.5"
+                                  }
+                              ]
+                          }
+                      }
+                  }
+              }
+          }
+      }
+      response = self.client.put(selfLink, data=json.dumps(payload), headers=headers)
+      self.waitForScaling(selfLink, 0)
+      total_time = int((time.time() - start_time) * 1000)
+      events.request_success.fire(request_type="fleet_scaling_down", name="fleet_scaling_down", response_time=total_time, response_length=0)
```

Comment:
@pm7h  Can you please run `pep8` util on your python code. For instance this lines too long:
 E501 line too long (132 > 79 characters)

Reply:
Done.

Reply:
Thanks @pm7h